.github/workflows/percy.yml

```diff
@@ -0,0 +1,50 @@
+name: Percy Visual Regression Tests
+
+on:
+  workflow_dispatch:
```

Comment:
Are we planning to only trigger this manually or shall we set up a cron job?

Reply:
Manually for the moment, until we come up with a proper CI strategy, that would include things like general automation coverage and such.